diff -urN leafnode-1.9.17/Makefile.in leafnode-1.9.17.pld/Makefile.in
--- leafnode-1.9.17/Makefile.in	Mon Sep  4 21:26:20 2000
+++ leafnode-1.9.17.pld/Makefile.in	Sat Sep 23 22:09:44 2000
@@ -123,17 +123,17 @@
 		-DVERSION=\"$(VERSION)\" $<
 
 install: liblnutil.a nntpd fetchnews texpire checkgroups applyfilter newsq
-	$(INSTALL) -d -o root -g 0 -m 755 $(INSTALLROOT)$(MANDIR)/man1
-	$(INSTALL) -d -o root -g 0 -m 755 $(INSTALLROOT)$(MANDIR)/man7
-	$(INSTALL) -d -o root -g 0 -m 755 $(INSTALLROOT)$(MANDIR)/man8
-	$(INSTALL) -d -o root -g bin -m 755 $(INSTALLROOT)$(BINDIR)
-	$(INSTALL) -d -o root -g bin -m 755 $(INSTALLROOT)$(USRDIR)
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) nntpd $(INSTALLROOT)$(BINDIR)/leafnode
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) fetchnews $(INSTALLROOT)$(BINDIR)/fetchnews
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) texpire $(INSTALLROOT)$(BINDIR)/texpire
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) checkgroups $(INSTALLROOT)$(BINDIR)/checkgroups
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) applyfilter $(INSTALLROOT)$(BINDIR)/applyfilter
-	$(INSTALL) -c -g news -m 755 -o news $(STRIP) newsq $(INSTALLROOT)$(USRDIR)/newsq
+	$(INSTALL) -d $(INSTALLROOT)$(MANDIR)/man1
+	$(INSTALL) -d $(INSTALLROOT)$(MANDIR)/man7
+	$(INSTALL) -d $(INSTALLROOT)$(MANDIR)/man8
+	$(INSTALL) -d $(INSTALLROOT)$(BINDIR)
+	$(INSTALL) -d $(INSTALLROOT)$(USRDIR)
+	$(INSTALL) -c $(STRIP) nntpd $(INSTALLROOT)$(BINDIR)/leafnode
+	$(INSTALL) -c $(STRIP) fetchnews $(INSTALLROOT)$(BINDIR)/fetchnews
+	$(INSTALL) -c $(STRIP) texpire $(INSTALLROOT)$(BINDIR)/texpire
+	$(INSTALL) -c $(STRIP) checkgroups $(INSTALLROOT)$(BINDIR)/checkgroups
+	$(INSTALL) -c $(STRIP) applyfilter $(INSTALLROOT)$(BINDIR)/applyfilter
+	$(INSTALL) -c $(STRIP) newsq $(INSTALLROOT)$(USRDIR)/newsq
 	$(INSTALL) -c -m 644 $(srcdir)/*.8 $(INSTALLROOT)$(MANDIR)/man8
 	if test $(PCRELIB) ; then $(INSTALL) -c -m 644 $(srcdir)/pcre/pcre.7 $(INSTALLROOT)$(MANDIR)/man7 ; fi
 	$(INSTALL) -c -m 644 $(srcdir)/*.1 $(INSTALLROOT)$(MANDIR)/man1
@@ -157,15 +157,10 @@
 				$${a}$${b}3 $${a}$${b}4 $${a}$${b}5 \
 				$${a}$${b}6 $${a}$${b}7 $${a}$${b}8 \
 				$${a}$${b}9 ; \
-			chown news:news $${a}$${b}0 $${a}$${b}1 $${a}$${b}2 \
-				$${a}$${b}3 $${a}$${b}4 $${a}$${b}5 \
-				$${a}$${b}6 $${a}$${b}7 $${a}$${b}8 \
-				$${a}$${b}9 ; \
 			done ; done
 	cp $(srcdir)/config.example $(INSTALLROOT)$(LIBDIR)
 	-chown root:news $(INSTALLROOT)$(LIBDIR)/config.example
 	-chmod 640 $(INSTALLROOT)$(LIBDIR)/config.example
-	chmod 2755 $(INSTALLROOT)$(SPOOLDIR)
 	@echo Edit /etc/inetd.conf to start $(BINDIR)/leafnode and restart inetd
 	@echo If you update from a leafnode version \< 1.9.3, do a \"make update\"
 
@@ -201,21 +196,13 @@
 		Makefile.in configure install-sh \
 		update.sh makerpm.sh leafnode-$(VERSION).lsm \
 		leafnode-$(VERSION)
-	chmod 644 leafnode-$(VERSION)/*
 	mkdir -p  leafnode-$(VERSION)/doc_german
-	chmod 755 leafnode-$(VERSION)/doc_german
 	mkdir -p leafnode-$(VERSION)/tools
-	chmod 755 leafnode-$(VERSION)/tools
 	cp -R doc_german/* leafnode-$(VERSION)/doc_german/
 	cp tools/* leafnode-$(VERSION)/tools/
-	chmod 755 leafnode-$(VERSION)/*sh
-	chmod 755 leafnode-$(VERSION)/configure
-	chmod 755 leafnode-$(VERSION)/tools/*
 	tar cf leafnode-$(VERSION).tar leafnode-$(VERSION)
 	mkdir -p leafnode-$(VERSION)/pcre
-	chmod 755 leafnode-$(VERSION)/pcre
 	cp pcre/* leafnode-$(VERSION)/pcre/
-	chmod 755 leafnode-$(VERSION)/pcre/perltest
 	tar cf leafnode-pcre-$(VERSION).tar leafnode-$(VERSION)
 	gzip -f9 leafnode-$(VERSION).tar
 	gzip -f9 leafnode-pcre-$(VERSION).tar
