diff -urN leafnode-1.9.4.orig/Makefile.in leafnode-1.9.4/Makefile.in
--- leafnode-1.9.4.orig/Makefile.in	Wed Jul 21 15:15:55 1999
+++ leafnode-1.9.4/Makefile.in	Wed Jul 21 15:44:28 1999
@@ -114,17 +114,17 @@
 		-DVERSION=\"$(VERSION)\" $<
 
 install: liblnutil.a nntpd fetchnews texpire checkgroups applyfilter newsq
-	$(INSTALL) -d -o root -g 0 -m 755 $(MANDIR)/man1
-	$(INSTALL) -d -o root -g 0 -m 755 $(MANDIR)/man7
-	$(INSTALL) -d -o root -g 0 -m 755 $(MANDIR)/man8
-	$(INSTALL) -d -o root -g bin -m 755 $(BINDIR)
-	$(INSTALL) -d -o root -g bin -m 755 $(USRDIR)
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) nntpd $(BINDIR)/leafnode
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) fetchnews $(BINDIR)/fetchnews
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) texpire $(BINDIR)/texpire
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) checkgroups $(BINDIR)/checkgroups
-	$(INSTALL) -c -g news -m 750 -o news $(STRIP) applyfilter $(BINDIR)/applyfilter
-	$(INSTALL) -c -g news -m 755 -o news $(STRIP) newsq $(USRDIR)/newsq
+	$(INSTALL) -d -m 755 $(MANDIR)/man1
+	$(INSTALL) -d -m 755 $(MANDIR)/man7
+	$(INSTALL) -d -m 755 $(MANDIR)/man8
+	$(INSTALL) -d -m 755 $(BINDIR)
+	$(INSTALL) -d -m 755 $(USRDIR)
+	$(INSTALL) -c -m 750 $(STRIP) nntpd $(BINDIR)/leafnode
+	$(INSTALL) -c -m 750 $(STRIP) fetchnews $(BINDIR)/fetchnews
+	$(INSTALL) -c -m 750 $(STRIP) texpire $(BINDIR)/texpire
+	$(INSTALL) -c -m 750 $(STRIP) checkgroups $(BINDIR)/checkgroups
+	$(INSTALL) -c -m 750 $(STRIP) applyfilter $(BINDIR)/applyfilter
+	$(INSTALL) -c -m 755 $(STRIP) newsq $(USRDIR)/newsq
 	cp $(srcdir)/*.8 $(MANDIR)/man8
 	if test $(PCRELIB) ; then cp $(srcdir)/pcre/pcre.7 $(MANDIR)/man7 ; fi
 	cp $(srcdir)/*.1 $(MANDIR)/man1
@@ -134,25 +134,14 @@
 	-mkdir -p $(SPOOLDIR)/message.id
 	-mkdir -p $(SPOOLDIR)/interesting.groups
 	-mkdir -p $(SPOOLDIR)/out.going
-	-chown news:news `dirname $(LOCKFILE)`
-	-chown news:news $(SPOOLDIR) $(LIBDIR)
-	-chown news:news $(SPOOLDIR)/leaf.node
-	-chown news:news $(SPOOLDIR)/message.id
-	-chown news:news $(SPOOLDIR)/interesting.groups
-	-chown news:news $(SPOOLDIR)/out.going
 	cd $(SPOOLDIR)/message.id ; for a in 0 1 2 3 4 5 6 7 8 9 ; do \
 		for b in 0 1 2 3 4 5 6 7 8 9 ; do \
 			mkdir -p $${a}$${b}0 $${a}$${b}1 $${a}$${b}2 \
 				$${a}$${b}3 $${a}$${b}4 $${a}$${b}5 \
 				$${a}$${b}6 $${a}$${b}7 $${a}$${b}8 \
 				$${a}$${b}9 ; \
-			chown news:news $${a}$${b}0 $${a}$${b}1 $${a}$${b}2 \
-				$${a}$${b}3 $${a}$${b}4 $${a}$${b}5 \
-				$${a}$${b}6 $${a}$${b}7 $${a}$${b}8 \
-				$${a}$${b}9 ; \
 			done ; done
 	cp $(srcdir)/config.example $(LIBDIR)
-	-chown root:news $(LIBDIR)/config.example
 	-chmod 640 $(LIBDIR)/config.example
 	chmod 2755 $(SPOOLDIR)
 	@echo Edit /etc/inetd.conf to start $(BINDIR)/leafnode and restart inetd
